### PR TITLE
Add logging to Scheduling Plugin

### PR DESCRIPTION
**Pull request by miaulalala**
## Summary

We currently don't have logging in the scheduling plugin at all. There is an ExceptionLoggerPlugin, but some debug info is always practical

To Do
- [x] Inject the logger
- [x] Fix failing tests if any

### CODE REVIEW

File: apps/dav/lib/CalDAV/Schedule/Plugin.php [hunk 1]
```diff
@@ -168,9 +168,9 @@ public function scheduleLocalDelivery(ITip\Message $iTipMessage):void {
 
 		// We only care when the message was successfully delivered locally
 		if ($iTipMessage->scheduleStatus !== '1.2;Message delivered locally') {
+			\OC::$server->get(\Psr\Log\LoggerInterface::class)->debug($iTipMessage->scheduleStatus);
```
Comment (tcitworld) [inline apps/dav/lib/CalDAV/Schedule/Plugin.php:171]: No extra context? Just the `scheduleStatus` is little information.

Comment (miaulalala) [reply apps/dav/lib/CalDAV/Schedule/Plugin.php:171]: this is a WIP

**Review by tcitworld — COMMENTED** [review]
`OCA\DAV\Tests\unit\CalDAV\Schedule\PluginTest` at least needs a change too.

**Review by julien-nc — APPROVED** [review]
Looks good

**Review by tcitworld — APPROVED** [review]
Needs a rebase, but code looks good.

**Pull request merged**